### PR TITLE
Move OC.Plugins, OC.msg and OC.Notification to the server bundle

### PR DESCRIPTION
**Pull request by ChristophWurst**
A different approach to #13938. Less fancy, but should work with Sinon. Let's see what the first commit does on CI before I continue this.

### CODE REVIEW

**Comment by rullzer** [comment]
Nope https://drone.nextcloud.com/nextcloud/server/15400/265

But I guess this is the unit tests missing something

**Comment by ChristophWurst** [comment]
Fixed with the latest fixup, I just forgot to rebuild.

File: core/src/OC/index.js [hunk 1]
```diff
@@ -34,4 +37,7 @@ export default {
 	ContactsMenu,
 	EventSource,
 	L10N,
+	msg,
```
Comment (skjnldsv) [inline core/src/OC/index.js]: PascalCase?

File: core/src/OC/msg.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+ *
+ * @namespace OC.msg
+ */
+export default {
+	/**
+	 * Displayes a "Saving..." message in the given message placeholder
+	 *
+	 * @param {Object} selector    Placeholder to display the message in
+	 */
+	startSaving: function (selector) {
```
Comment (skjnldsv) [inline core/src/OC/msg.js]: What and where is this used for :thinking:

File: core/src/OC/index.js (hunk 1 above)
Comment (ChristophWurst) [reply core/src/OC/index.js]: This would break backwards compatibility

File: core/src/OC/msg.js (hunk 2 above)
Comment (ChristophWurst) [reply core/src/OC/msg.js]: I haven't checked. I don't plan on removing or changing the APIs right now

**Review by MorrisJobke — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Pull request merged**